package.json

```diff
@@ -1,29 +1,40 @@
 {
   "name": "koa-redis",
-  "description": "koa session with redis",
-  "repository": "dead-horse/koa-redis",
+  "description": "koa session with redis using koa-generic-session",
+  "repository": "koajs/koa-redis",
+  "bugs": {
+    "url": "https://github.com/koajs/koa-redis/issues"
+  },
+  "homepage": "https://github.com/koajs/koa-redis",
   "version": "1.0.1",
-  "keywords": ["koa", "middleware", "session", "redis"],
-  "files": "index.js",
+  "main": "./index.js",
+  "scripts": {
+    "test": "istanbul cover node_modules/mocha/bin/_mocha -- \"test/**/*.test.js\"",
+    "test-only": "mocha \"test/**/*.test.js\""
+  },
+  "keywords": [
+    "koa",
+    "middleware",
+    "session",
+    "redis"
+  ],
   "author": "dead_horse <dead_horse@qq.com>",
   "dependencies": {
-    "co-redis": "~1.1.0",
+    "co-redis": "2.0.*",
```

Comment:
Fixed in https://github.com/DaAwesomeP/koa-redis/commit/a64752f72381b4a9a6d0a404c8969bf3bc39c6b2